### PR TITLE
fix build break in transformer debug dump

### PR DESCRIPTION
**Pull request by fs-eire**
### Description

Fix build break in transformer debug dump introduced in #13954.

### CODE REVIEW

**Review by prathikr — APPROVED** [review]
LGTM

**Review by snnn — APPROVED** [review]

**Comment by snnn** [comment]
Please wait until the "Post Merge Jobs" is completed. It is a new daily pipeline that was just added recently.

**Pull request merged**
